crates/router/src/connector/stripe.rs

```diff
@@ -1718,7 +1718,10 @@ impl
         errors::ConnectorError,
     > {
         let response = res.response;
-        router_env::logger::info!(connector_response=?response);
+
+        event_builder.map(|event| event.set_response_body(&serde_json::json!({"connector_response": response.escape_ascii().to_string(), "status_code": res.status_code})));
```

Comment:
this is like logging raw request bytes,
this might expose secret data,

you can just do 
```rust
event_builder.map(|event| event.set_response_body(&serde_json::json!({"connector_response_type": "file", "status_code": res.status_code})));
```